tests/UI/pages/BO/catalog/productsV2/add/index.ts

```diff
@@ -174,7 +174,7 @@ class CreateProduct extends BOBasePage {
    * @returns {Promise<string>}
```

Comment:
should we also move pages from productV2 to product ?

Reply:
I will do that after upgrade of functional part. It's just sanity.